tests/test_managed_allow_list.py

```diff
@@ -1,160 +1,101 @@
+import os
+
 import ape
 import pytest
-
-RITUAL_ID = 0
-ADMIN_CAP = 5
-ERC20_SUPPLY = 10**24
-FEE_RATE = 42
+from eth_account.messages import encode_defunct
+from web3 import Web3
 
 
 @pytest.fixture(scope="module")
-def deployer(accounts):
-    return accounts[0]
+def initiator(accounts):
+    initiator_index = 1
+    assert len(accounts) >= initiator_index
```

Comment:
I know you are being complete here 😄 , but our ape-config generates 40 test accounts. I'm ok with this function not checking the index, but that's up to you.

In any case, as part of https://github.com/nucypher/nucypher-contracts/issues/284 we will likely make these fixtures common, so fine to leave this for now.

Reply:
more like I forgot to remove it 
✔️